src/itest/java/uk/gov/companieshouse/charges/data/steps/ChargesApiSteps.java

```diff
@@ -64,17 +64,16 @@ public class ChargesApiSteps {
 
     String companyNumber = "08124207";
     String chargeId = "AbRiNTU3NjNjZWI1Y2YxMzkzYWY3MzQ0YzVlOTg4ZGVhZTBkYWI4Ng==";
-    String insolvency_cases_happy_path_input = "Insolvency_cases_Happy_Path_input";
-    String invalid_payload = "Invalid_payload";
-    String x_request_value = "5234234234";
-    String x_request_id = "x-request-id";
+    String insolvencyCasesHappyPathInput = "Insolvency_cases_Happy_Path_input";
```

Comment:
These should all really be `private static final` strings with variable names as upper case/snake case

Reply:
Addressed in most recent commit